libs/ui-components/src/hooks/useAppContext.tsx

```diff
@@ -36,6 +36,12 @@ export const appRoutes = {
 export type NavLinkFC = React.FC<{ to: string; children: (props: { isActive: boolean }) => React.ReactNode }>;
 export type PromptFC = React.FC<{ message: string }>;
 
+export type JSONPatch<V = unknown> = {
```

Comment:
We could use the type `PatchRequest` generated by the OpenApi spec, instead of defining our own JSONPatch. Doesn't seem to make a difference the `any` vs `unknown` for the value.

For using it as a single instance, we can do `PatchRequest[0]`

Reply:
I didn't know about `PatchRequest[0]` which is nice! 
And I didn't need it in the end :) 

JSONPatch type is now removed.